frontend/src/container/NewDashboard/DashboardDescription/index.tsx

```diff
@@ -283,7 +283,7 @@ function DashboardDescription(props: DashboardDescriptionProps): JSX.Element {
 						className="dashboard-btn"
 						onClick={(): void => goToListPage()}
 					>
-						Dashboard /
+						Dashboard
```

Comment:
unexpected and its brokes the UI - 

Before:
<img width="315" alt="Screenshot 2024-07-26 at 1 41 47 PM" src="https://github.com/user-attachments/assets/412bb712-4814-4162-9566-bb0fa7125db6">

After:
<img width="315" alt="image" src="https://github.com/user-attachments/assets/6b25f0af-abd7-40f5-a6cf-f0b1bb1c3d5c">

Reply:
Thanks for noticing, will update